packages/terra-modal/lib/Modal.scss

```diff
@@ -0,0 +1,34 @@
+@import '~terra-mixins';
+
+.terra-Modal {
+  background-color: #fff;
```

Comment:
Should we enforce the use of terra-legacy-theme colors? 
https://github.com/cerner/terra-core/blob/master/packages/terra-legacy-theme/src/_palette.scss#L67